build.gradle

```diff
@@ -24,7 +24,8 @@ buildscript {
         opensearch_build = version_tokens[0] + '.0'
 
         common_utils_version = System.getProperty("common_utils.version", '3.0.0.0-SNAPSHOT')
-        kafka_version  = '3.0.2'
+        kafka_version  = '3.4.0'
+        kafkaVersion   = '3.4.0'
```

Comment:
Unused, lets remove

Reply:
kafka_version  = '3.4.0' //for kafka client 
 kafkaVersion   = '3.4.0' // hacked spring kafka test framework or else it only support 3.3.2